DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/AdminFieldsController.cs

```diff
@@ -146,6 +147,107 @@ BulkAdminFieldAnswersViewModel model
             );
         }
 
+        [HttpGet]
+        [Route("{customisationId:int}/AdminFields/Add/New")]
+        public IActionResult AddAdminFieldNew(int customisationId)
+        {
+            TempData.Clear();
+
+            var model = new AddAdminFieldViewModel(customisationId);
+            var addAdminFieldData = new AddAdminFieldData(model);
+            var id = addAdminFieldData.Id;
+
+            Response.Cookies.Append(
+                AddPromptCookieName,
+                id.ToString(),
+                new CookieOptions
+                {
+                    Expires = CookieExpiry
+                }
+            );
+            TempData.Set(addAdminFieldData);

```

Comment:
Could pull this into a private method to make it easier to glance over the controller?